plugins/reports/api/reports.js

```diff
@@ -304,7 +348,7 @@ var metrics = {
                         var monthName = moment.localeData().monthsShort(moment([0, startDate.getMonth()]), "");
 
                         report.date = startDate.getDate() + " " + monthName;
-                        report.period = "yesterday";
+                        report.period = "hour";
```

Comment:
I think we intentionally provide yesterday's data, as today will not be a full day, and user's usually want report of time that has passed

Reply:
right, should be "yesterday"